lib/src/bloc_provider.dart

```diff
@@ -228,11 +228,16 @@ class _Inherited<BlocType extends Bloc> extends InheritedWidget {
 
   static BlocType of<BlocType extends Bloc>(BuildContext context) {
     Type typeOf<T>() => T;
-    return (context
-            .ancestorInheritedElementForWidgetOfExactType(
-                typeOf<_Inherited<BlocType>>())
-            .widget as _Inherited<BlocType>)
-        .bloc;
+    final widget = context
+        .ancestorInheritedElementForWidgetOfExactType(
+            typeOf<_Inherited<BlocType>>())
+        .widget as _Inherited<BlocType>;
+    if (widget == null) {
+      throw ArgumentError(
+          // ignore: lines_longer_than_80_chars
```

Comment:
I've deleted this ignore by using adjacent string literals: https://github.com/mono0926/bloc_provider/commit/3b3b4efbb97fe6a2bed67eaf1aacefbd18c568a0